runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -11824,6 +11824,10 @@ J9::Power::CodeGenerator::inlineDirectCall(TR::Node *node, TR::Register *&result
       resultReg = inlineSinglePrecisionFP(node, TR::InstOpCode::fsqrts, cg);
       return true;
       }
+   else if (OMR::Power::CodeGenerator::inlineDirectCall(node, resultReg))
+      {
+      return true;
+      }
```

Comment:
This is not the right way to call the underlying OMR API as there could be other extensions on top of on top of Power extension point in OMR. The proper way to call this is via a connector, i.e. 

```cpp
OMR::CodeGeneratorConnector::inlineDirectCall(node, resultReg)
```

Reply:
We still need a fix here. Once it's done this should be ready.

Reply:
Thanks, Filip.  Revised in change https://github.com/eclipse/openj9/pull/3086/commits/545a95fa85e2d80a2492b2bc3bd60e955da9806a